x-pack/filebeat/module/fortinet/clientendpoint/test/generated.log-expected.json

```diff
@@ -0,0 +1,5702 @@
+        ],
+        "source.port": 2638,
+        "tags": [
+            "fortinet.clientendpoint",
+            "forwarded"
+        ],
+        "user.name": "onev"
+    },
+    {
+        "@timestamp": "2020-04-09T19:22:51.000Z",
+        "destination.ip": [
+            "10.11.200.161"
+        ],
+        "destination.port": 4665,
+        "event.action": "deny",
+        "event.code": "http",
+        "event.dataset": "fortinet.clientendpoint",
+        "event.module": "fortinet",
+        "event.original": "April 9 17:22:51 uatDuis2964.test proto=udp service=http status=deny src=10.183.202.41 dst=10.11.200.161 src_port=4470 dst_port=4665 server_app=inimve pid=4243 app_name=antium traff_direct=Cice block_count=513 logon_user=iusmodt@doloreeu3553.www5.home msg=unknown",
+        "event.outcome": "Failure",
+        "fileset.name": "clientendpoint",
+        "host.name": "uatDuis2964.test",
+        "input.type": "log",
+        "log.offset": 1355,
+        "network.direction": "Cice",
+        "network.protocol": "udp",
+        "observer.product": "FortiClient",
+        "observer.type": "Anti-Virus",
+        "observer.vendor": "Fortinet",
+        "process.pid": 4243,
+        "related.ip": [
+            "10.183.202.41",
+            "10.11.200.161"
+        ],
+        "related.user": [
+            "iusmodt"
+        ],
+        "rsa.counters.dclass_c1": 513,
+        "rsa.counters.dclass_c1_str": "block_count",
+        "rsa.internal.messageid": "http",
+        "rsa.investigations.ec_outcome": "Failure",
+        "rsa.investigations.ec_subject": "NetworkComm",
+        "rsa.investigations.ec_theme": "ALM",
+        "rsa.misc.action": [
+            "deny"
+        ],
+        "rsa.misc.result": "unknown",
+        "rsa.network.alias_host": [
+            "uatDuis2964.test"
+        ],
+        "rsa.network.domain": "doloreeu3553.www5.home",
+        "rsa.network.network_service": "http",
+        "rsa.time.event_time": "2020-04-09T19:22:51.000Z",
+        "server.domain": "doloreeu3553.www5.home",
+        "service.type": "fortinet",
+        "source.ip": [
+            "10.183.202.41"
+        ],
+        "source.port": 4470,
+        "tags": [
+            "fortinet.clientendpoint",
+            "forwarded"
+        ],
+        "user.name": "iusmodt"
+    },
+    {
+        "@timestamp": "2020-04-24T02:25:25.000Z",
+        "destination.ip": [
+            "10.214.225.125"
+        ],
+        "destination.port": 2121,
+        "event.action": "deny",
+        "event.code": "pop3",
+        "event.dataset": "fortinet.clientendpoint",
+        "event.module": "fortinet",
+        "event.original": "April 24 00:25:25 reetdolo2770.www5.local proto=tcp service=pop3 status=deny src=10.12.44.169 dst=10.214.225.125 src_port=5710 dst_port=2121 server_app=inBCSedu pid=5722 app_name=tanimi traff_direct=nimadmin block_count=6499 logon_user=uam@temq1198.internal.example msg=success",
+        "event.outcome": "Failure",
```

Comment:
this looks like it's always set to `Failure`? `msg:failure`, `msg:success`, and `msg:unknown` all appear to be `Failure`

Reply:
This is something weird this parser does. It hardcodes `ec_outcome` to `Failure` (mapped to `event.outcome`). Then captures the `msg` in `result`, which is mapped to `rsa.misc.result`.

I'll see if that can be fixed by giving `result` priority over `ec_outcome` to populate `event.outcome`, but need to be careful as it might break other parsers.